pom.xml

```diff
@@ -224,6 +224,12 @@ under the License.
 				<version>3.2.1</version>
 			</dependency>
 
+			<dependency>
+				<groupId>org.apache.commons</groupId>
+				<artifactId>commons-math3</artifactId>
+				<version>3.5</version>
+			</dependency>
```

Comment:
Sorry, they are actually located in `flink-dist/src/main/flink-bin`. But I'll update them when I merge the PR.